core/tests/puppeteer-acceptance-tests/specs/topic-manager/access-dashboard-using-profile-dropdown.spec.ts

```diff
@@ -0,0 +1,174 @@
+// Copyright 2025 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance test from CUJv3 Doc
+ * https://docs.google.com/document/d/1D7kkFTzg3rxUe3QJ_iPlnxUzBFNElmRkmAWss00nFno/
+ *
+ * TM.TD. Access dashboard using Profile Dropdown.
+ */
+
+import testConstants from '../../utilities/common/test-constants';
+import {UserFactory} from '../../utilities/common/user-factory';
+import {CurriculumAdmin} from '../../utilities/user/curriculum-admin';
+import {ExplorationEditor} from '../../utilities/user/exploration-editor';
+import {LoggedInUser} from '../../utilities/user/logged-in-user';
+import {TopicManager} from '../../utilities/user/topic-manager';
+
+const ROLES = testConstants.Roles;
+
+describe('Topic Manager', function () {
+  let topicManager: TopicManager & LoggedInUser;
+  let curriculumAdmin: CurriculumAdmin &
+    ExplorationEditor &
+    TopicManager &
+    LoggedInUser;
+
+  beforeAll(async function () {
+    curriculumAdmin = await UserFactory.createNewUser(
+      'curriculumAdm',
+      'curriculum_adm@example.com',
+      [ROLES.CURRICULUM_ADMIN]
+    );
+
+    const explorationId =
+      await curriculumAdmin.createAndPublishExplorationWithCards(
+        'Solving problems without a calculator',
+        'Mathematics'
+      );
+    await curriculumAdmin.createAndPublishTopic(
+      'Arithmetic Operations',
+      'Addition',
+      'Addition'
+    );
+    await curriculumAdmin.addStoryToTopic(
+      'The Broken Calculator',
+      'the-broken-calculator',
+      'Arithmetic Operations'
+    );
+    await curriculumAdmin.addChapter(
+      'Solving problems without a calculator',
+      explorationId
+    );
+    await curriculumAdmin.createAndPublishClassroom(
+      'Maths',
+      'maths',
+      'Arithmetic Operations'
+    );
+
+    // Create more topics and skills.
+    await curriculumAdmin.createTopic('Whole Numbers', 'whole-numbers');
+    await curriculumAdmin.createSkillFromSkillsDashboard(
+      'Subtraction',
+      'Review Material for Subtraction'
+    );
+    await curriculumAdmin.createSkillFromSkillsDashboard(
+      'Word Problems',
+      'Review Material for Word Problems'
+    );
+
+    // Create topic manager user.
+    topicManager = await UserFactory.createNewUser(
+      'topicManager',
+      'topic_manager@example.com',
+      [ROLES.TOPIC_MANAGER],
+      'Arithmetic Operations'
+    );
+  }, 600000);
```

Comment:
why doesn't the default work here? ditto for the other cases of this

For reference, here's where the default is defined

https://github.com/oppia/oppia/blob/0f7d947f7dab3c9001147f95e40f2d9923d67137/core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts#L256

Reply:
It's because the setup takes longer than default timeout of 5 minutes.